### PR TITLE
Projecct 5 - Cluedo

### PR DESCRIPTION
**Pull request by Jenfi**
I was struggling a lot with the javascript. 

I also found that the instructions were quite difficult to understand, which made it even a bit more tricky. 

I decided not to make the site responsive for now, just to save time.

### CODE REVIEW

**Review by ericaglimsholt — CHANGES_REQUESTED** [review]
Hi! Can you add the Netlify link to your PR?

**Comment by Jenfi** [comment]
> Hi! Can you add the Netlify link to your PR?

Hi Erica,

I linked the wrong Netlify by mistake, pls note correct link:
https://jenniferfick-cluedo-project.netlify.com/

File: code/style.css [hunk 1]
```diff
@@ -1,37 +1,105 @@
...
-  justify-content: space-around;
+  justify-content: space-evenly;
   align-items: center;
 }
 
 .killer-deck {
   width: 250px;
   height: 400px;
   background: #000;
+  background: url("questionmark.png") center no-repeat;
```
Comment (ericaglimsholt) [inline code/style.css:87]: This is so cleaver! It looks really nice

File: code/clue.js [hunk 2]
```diff
@@ -1,64 +1,279 @@
...
+}
+
+const missScarlet = {
+  firstName: "Cassandra",
+  lastName: "Scarlet",
+  color: "red",
+  nameCol: "beige",
+  description: "A movie star who will do anything to stay in the spotlight",
+  age: 25,
+  image: "scarlett.png",
```
Comment (ericaglimsholt) [inline code/clue.js:30]: How come the image for missScarlet isn't in the assets folder like the other ones?

File: code/style.css [hunk 3]
```diff
@@ -1,37 +1,105 @@
...
   margin: 0;
-  font-family: "Courier New", Courier, monospace;
+  background: silver;
+  font-family: 'Roboto Mono', monospace, Georgia;
+}
+
+h1{
+text-align: center;
+font-family:'Alfa Slab One', cursive, Georgia;
+font-size: 50px;
```
Comment (ericaglimsholt) [inline code/style.css:10]: Indent

File: code/style.css [hunk 4]
```diff
@@ -1,37 +1,105 @@
...
+}
+
+h2{
+  text-align: center;
+    padding: 7px;
```
Comment (ericaglimsholt) [inline code/style.css:15]: Unindent

File: code/style.css [hunk 5]
```diff
@@ -1,37 +1,105 @@
...
+
+#killerAge{
+  display: inline;
+  font-size: 16px;
+}
+
+#killerOccupation{
+    font-size: 14px;
+    margin-top: 20px;
+  }
```
Comment (ericaglimsholt) [inline code/style.css:42]: Unindent

**Review by ericaglimsholt — APPROVED** [review]
Nice work on the Cluedo project! Seems like you manage Javascript pretty well! Looking forward to see the next project

**Pull request closed without merging**